sdk/core/Azure.Core.TestFramework/src/RecordedTestBase.cs

```diff
@@ -134,6 +134,24 @@ public void StopLoggingEvents()
             Logger = null;
         }
 
+        [OneTimeSetUp]
+        public void UpdateDefaultPollingTime()
```

Comment:
Is there any point in restoring the original Polling Time as part of the tear down?

Reply:
Another question: I don't think we currently have any test scenarios affected by that, but should we be concerned about adding a test flag to skip this step? This can be hard to accomplish since we'll have tests running in parallel.

Reply:
I'm not very comfortable with this approach. As you've said tests can be running in parallel and changing this setting might affect non-playback tests.

We have a common pattern where a single test is changed to record using a ctor parameter. If in that situation someone accidentally runs recorded tests along with the ones running live they'll flood the service with requests.

Reply:
Maybe it's OK to do on CI where we know the mode is global.

Reply:
I wonder if we go a different way and instrument operations returned by clients the same way we instrument clients themselves. This way we can intercept the WaitForComplion call and pass 0 timeout to the base implementation.